script.js

```diff
@@ -1,4 +1,109 @@
 var main = function (input) {
```

Comment:
consider refactoring your main function further into smaller functions, i've given some pointers on how the main function can be made less repetitive which would help cut down the length but def still consider breaking it up further.

your game has distinct stages:
1. player rolls dice
2. player chooses order
3. once both players have done so, compare score and output winner

this provides a very nice starting point for how to refactor